blockly-rtc/src/UserDataManager.js

```diff
@@ -49,7 +50,24 @@ export default class UserDataManager {
     } else {
       this.pollServer_();
     };
-};
+    if (this.getUserDisconnects) {
+      this.getUserDisconnects(this.disposeMarker_.bind(this));
```

Comment:
Won't this always be null here?

Reply:
The current flow is that `enablePresence()` assigns `this.getUserDisconnects`. And then `start()` is called which calls the function.

Another approach is to assign and call the function `this.getUserDisconnects` in `enablePresence` and remove L53-54 from `start`.

I am open and happy with either option.

Reply:
Correction, when I say `enablePresence` above, I am actually referring to `setPresenceHandlers`